crates/egui-winit/src/lib.rs

```diff
@@ -865,70 +865,85 @@ impl State {
     }
 }
 
-/// Update the given viewport info with the current state of the window.
-///
-/// Call before [`State::take_egui_input`].
-pub fn update_viewport_info(
-    viewport_info: &mut ViewportInfo,
-    egui_ctx: &egui::Context,
-    window: &Window,
-) {
-    crate::profile_function!();
+pub fn math_inner_rect(window: &Window, pixels_per_point: Option<f32>) -> Option<Rect> {
+    let pixels_per_point = match pixels_per_point {
+        Some(pixels_per_point) => pixels_per_point,
+        None => window.scale_factor() as f32,
+    };
```

Comment:
This is never called with `None`, so let's simplify

```suggestion
pub fn math_inner_rect(window: &Window, pixels_per_point: f32) -> Option<Rect> {
```